### ISSUE TITLE
deliver finished successfully, but the package is missing

### ISSUE THREAD

**Issue by trollderius** [fastlane/fastlane#18554]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
fastlane "deliver" finished successfully without any errors and shows the message:
"Succesffully uploaded package to App Store Connect. It might take a few minutes until it's visible online"
But the binary never showed on appstoreconnect or transporter.

##### Command executed

```
lane :ios_deliver_skip do
  deliver(
    force: true,
    automatic_release: true,
    skip_screenshots: true,
    skip_metadata: true
  )
end
```

`fastlane ios_deliver_skip`

##### Complete output when running fastlane, including the stack trace and command used
<details>
  <pre>
[✔] 🚀 
[01:59:47]: fastlane detected a Gemfile in the current directory
[01:59:47]: However, it seems like you didn't use `bundle exec`
[01:59:47]: To launch fastlane faster, please use
[01:59:47]: 
[01:59:47]: $ bundle exec fastlane ios_deliver_skip
[01:59:47]: 
[01:59:47]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
Your Gemfile lists the gem fastlane-plugin-upgrade_super_old_xcode_project (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
[01:59:48]: Plugin 'UpgradeSuperOldXcodeProject' overwrites already loaded action 'upgrade_super_old_xcode_project'
+---------------------------------------------+---------+---------------------------------------------+
|                                            Used plugins                                             |
+---------------------------------------------+---------+---------------------------------------------+
| Plugin                                      | Version | Action                                      |
+---------------------------------------------+---------+---------------------------------------------+
| fastlane-plugin-upgrade_super_old_xcode_pr  | 0.0.2   | upgrade_super_old_xcode_project             |
| oject                                       |         |                                             |
| fastlane-plugin-huawei_appgallery_connect   | 1.0.15  | huawei_appgallery_connect_submit_for_revie  |
|                                             |         | w huawei_appgallery_connect                 |
|                                             |         | huawei_appgallery_connect_get_app_info      |
+---------------------------------------------+---------+---------------------------------------------+

[01:59:48]: Driving the lane 'ios_deliver_skip' 🚀
[01:59:48]: ---------------------
[01:59:48]: --- Step: deliver ---
[01:59:48]: ---------------------
[01:59:48]: Successfully loaded '/Users/troll/Documents/click-to-vote/cordova/fastlane/Deliverfile' 📄

+---------------+-------------------------------------------------------------------+
|                   Detected Values from './fastlane/Deliverfile'                   |
+---------------+-------------------------------------------------------------------+
| release_notes | {"default"=>"Fixed minor bugs", "ca"=>"Fixed minor bugs",         |
|               | "cs"=>"Fixed minor bugs", "da"=>"Fixed minor bugs",               |
|               | "de-DE"=>"Fixed minor bugs", "el"=>"Fixed minor bugs",            |
|               | "en-US"=>"Fixed minor bugs", "es-ES"=>"Fixed minor bugs",         |
|               | "fi"=>"Fixed minor bugs", "fr-FR"=>"Fixed minor bugs",            |
|               | "hi"=>"Fixed minor bugs", "hr"=>"Fixed minor bugs", "hu"=>"Fixed  |
|               | minor bugs", "id"=>"Fixed minor bugs", "it"=>"Fixed minor bugs",  |
|               | "ja"=>"Fixed minor bugs", "ko"=>"Fixed minor bugs", "ms"=>"Fixed  |
|               | minor bugs", "nl-NL"=>"Fixed minor bugs", "no"=>"Fixed minor      |
|               | bugs", "pl"=>"Fixed minor bugs", "pt-PT"=>"Fixed minor bugs",     |
|               | "ro"=>"Fixed minor bugs", "ru"=>"Fixed minor bugs", "sk"=>"Fixed  |
|               | minor bugs", "sv"=>"Fixed minor bugs", "th"=>"Fixed minor bugs",  |
|               | "tr"=>"Fixed minor bugs", "uk"=>"Fixed minor bugs", "vi"=>"Fixed  |
|               | minor bugs", "zh-Hans"=>"Fixed minor bugs", "zh-Hant"=>"Fixed     |
|               | minor bugs"}                                                      |
+---------------+-------------------------------------------------------------------+

[01:59:48]: Login to App Store Connect (****)
[01:59:49]: Login successful

+--------------------------------------+------------------------+
|                    deliver 2.180.1 Summary                    |
+--------------------------------------+------------------------+
| force                                | true                   |
| automatic_release                    | true                   |
| skip_screenshots                     | true                   |
| skip_metadata                        | true                   |
| screenshots_path                     | ./fastlane/screenshots |
| metadata_path                        | ./fastlane/metadata    |
| app_version                          | 1                      |
| username                             | **** |
| app_identifier                       | me.clicktovote.app     |
| ipa                                  | click-to-vote.ipa      |
| platform                             | ios                    |
| edit_live                            | false                  |
| use_live_version                     | false                  |
| skip_binary_upload                   | false                  |
| skip_app_version_update              | false                  |
| overwrite_screenshots                | false                  |
| submit_for_review                    | false                  |
| reject_if_possible                   | false                  |
| phased_release                       | false                  |
| reset_ratings                        | false                  |
| run_precheck_before_submit           | true                   |
| precheck_default_rule_level          | warn                   |
| release_notes.default                | Fixed minor bugs       |
| release_notes.ca                     | Fixed minor bugs       |
| release_notes.cs                     | Fixed minor bugs       |
| release_notes.da                     | Fixed minor bugs       |
| release_notes.de-DE                  | Fixed minor bugs       |
| release_notes.el                     | Fixed minor bugs       |
| release_notes.en-US                  | Fixed minor bugs       |
| release_notes.es-ES                  | Fixed minor bugs       |
| release_notes.fi                     | Fixed minor bugs       |
| release_notes.fr-FR                  | Fixed minor bugs       |
| release_notes.hi                     | Fixed minor bugs       |
| release_notes.hr                     | Fixed minor bugs       |
| release_notes.hu                     | Fixed minor bugs       |
| release_notes.id                     | Fixed minor bugs       |
| release_notes.it                     | Fixed minor bugs       |
| release_notes.ja                     | Fixed minor bugs       |
| release_notes.ko                     | Fixed minor bugs       |
| release_notes.ms                     | Fixed minor bugs       |
| release_notes.nl-NL                  | Fixed minor bugs       |
| release_notes.no                     | Fixed minor bugs       |
| release_notes.pl                     | Fixed minor bugs       |
| release_notes.pt-PT                  | Fixed minor bugs       |
| release_notes.ro                     | Fixed minor bugs       |
| release_notes.ru                     | Fixed minor bugs       |
| release_notes.sk                     | Fixed minor bugs       |
| release_notes.sv                     | Fixed minor bugs       |
| release_notes.th                     | Fixed minor bugs       |
| release_notes.tr                     | Fixed minor bugs       |
| release_notes.uk                     | Fixed minor bugs       |
| release_notes.vi                     | Fixed minor bugs       |
| release_notes.zh-Hans                | Fixed minor bugs       |
| release_notes.zh-Hant                | Fixed minor bugs       |
| ignore_language_directory_validation | false                  |
| precheck_include_in_app_purchases    | true                   |
+--------------------------------------+------------------------+

[01:59:52]: Making sure the latest version on App Store Connect matches '1'...
[01:59:52]: '1' is the latest version on App Store Connect
[01:59:52]: Detected languages: ["default", "ca", "cs", "da", "de-DE", "el", "en-US", "es-ES", "fi", "fr-FR", "hi", "hr", "hu", "id", "it", "ja", "ko", "ms", "nl-NL", "no", "pl", "pt-PT", "ro", "ru", "sk", "sv", "th", "tr", "uk", "vi", "zh-Hans", "zh-Hant"]
[01:59:52]: Uploading binary to App Store Connect
[01:59:52]: Fetching password for transporter from environment variable named `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD`
[01:59:52]: Going to upload updated app to App Store Connect
[01:59:52]: This might take a few minutes. Please don't interrupt the script.
[02:00:24]: iTunes Transporter successfully finished its job
[02:00:24]: ------------------------------------------------------------------------------------------------------------------
[02:00:24]: --- Successfully uploaded package to App Store Connect. It might take a few minutes until it's visible online. ---
[02:00:24]: ------------------------------------------------------------------------------------------------------------------
[02:00:24]: Finished the upload to App Store Connect
[02:00:24]: Running precheck before submitting to review, if you'd like to disable this check you can set run_precheck_before_submit to false
[02:00:24]: Running precheck 👮‍♀️ 👮

+--------------------------+------------------------+
|           Summary for precheck 2.180.1            |
+--------------------------+------------------------+
| default_rule_level       | warn                   |
| include_in_app_purchases | true                   |
| app_identifier           | me.clicktovote.app     |
| username                 | **** |
| platform                 | ios                    |
| use_live                 | false                  |
+--------------------------+------------------------+

[02:00:24]: Checking app for precheck rule violations
[02:00:31]: Reading in-app purchases. If you have a lot, this might take a while
[02:00:31]: You can disable IAP checking by setting the `include_in_app_purchases` flag to `false`
[02:00:31]: Done reading in-app purchases
[02:00:32]: ✅  Passed: No negative  sentiment
[02:00:32]: ✅  Passed: No placeholder text
[02:00:32]: ✅  Passed: No mentioning  competitors
[02:00:32]: ✅  Passed: No future functionality promises
[02:00:32]: ✅  Passed: No words indicating test content
[02:00:32]: ✅  Passed: No curse words
[02:00:32]: ✅  Passed: No words indicating your IAP is free
[02:00:32]: ✅  Passed: Incorrect, or missing copyright date
[02:00:49]: ✅  Passed: No broken urls
[02:00:49]: precheck 👮‍♀️ 👮  finished without detecting any potential problems 🛫

+------+---------+-------------+
|       fastlane summary       |
+------+---------+-------------+
| Step | Action  | Time (in s) |
+------+---------+-------------+
| 1    | deliver | 61          |
+------+---------+-------------+

[02:00:49]: fastlane.tools finished successfully 🎉
  </pre>
</details>

### Environment

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.7                                     |
| Ruby                        | 2.7.3                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.24.3 (Apple Git-128)          |
| Installation Source         | ~/.rvm/gems/ruby-2.7.2/bin/fastlane         |
| Host                        | Mac OS X 10.15.7 (19H524)                   |
| Ruby Lib Dir                | /usr/local/Cellar/ruby@2.7/2.7.3/lib        |
| OpenSSL Version             | OpenSSL 1.1.1k  25 Mar 2021                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 12.4                                        |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
lane :ios_deliver_skip do
  deliver(
    force: true,
    automatic_release: true,
    skip_screenshots: true,
    skip_metadata: true
  )
end
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier ENV["APP_IDENTIFIER"]
package_name ENV["APP_IDENTIFIER"]

# ANDROID
json_key_file "config/google-service-account.json"

# IOS
apple_id "****"
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.180.1 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                          | Version | Update-Status |
| ----------------------------------------------- | ------- | ------------- |
| fastlane-plugin-upgrade_super_old_xcode_project | 0.0.2   | ✅ Up-To-Date  |
| fastlane-plugin-huawei_appgallery_connect       | 1.0.15  | ✅ Up-To-Date  |
| fastlane-plugin-upgrade_super_old_xcode_project | 0.0.2   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                             | Version      |
| ----------------------------------------------- | ------------ |
| did_you_mean                                    | 1.4.0        |
| slack-notifier                                  | 2.3.2        |
| atomos                                          | 0.1.3        |
| CFPropertyList                                  | 3.0.3        |
| claide                                          | 1.0.3        |
| colored2                                        | 3.1.2        |
| nanaimo                                         | 0.3.0        |
| xcodeproj                                       | 1.19.0       |
| rouge                                           | 2.0.7        |
| xcpretty                                        | 0.3.0        |
| terminal-notifier                               | 2.0.0        |
| unicode-display_width                           | 1.7.0        |
| terminal-table                                  | 1.8.0        |
| plist                                           | 3.6.0        |
| public_suffix                                   | 4.0.6        |
| addressable                                     | 2.7.0        |
| multipart-post                                  | 2.0.0        |
| word_wrap                                       | 1.0.0        |
| tty-screen                                      | 0.8.1        |
| tty-cursor                                      | 0.7.1        |
| tty-spinner                                     | 0.9.3        |
| artifactory                                     | 3.0.15       |
| babosa                                          | 1.0.4        |
| colored                                         | 1.2          |
| highline                                        | 1.7.10       |
| commander-fastlane                              | 4.4.6        |
| excon                                           | 0.79.0       |
| faraday-net_http                                | 1.0.1        |
| ruby2_keywords                                  | 0.0.4        |
| faraday                                         | 1.3.0        |
| unf_ext                                         | 0.0.7.7      |
| unf                                             | 0.1.4        |
| domain_name                                     | 0.5.20190701 |
| http-cookie                                     | 1.0.3        |
| faraday-cookie_jar                              | 0.0.7        |
| faraday_middleware                              | 1.0.0        |
| fastimage                                       | 2.2.3        |
| gh_inspector                                    | 1.1.3        |
| mini_magick                                     | 4.11.0       |
| naturally                                       | 2.2.1        |
| rubyzip                                         | 2.3.0        |
| security                                        | 0.1.3        |
| xcpretty-travis-formatter                       | 1.0.1        |
| dotenv                                          | 2.7.6        |
| bundler                                         | 2.2.15       |
| simctl                                          | 1.6.8        |
| jwt                                             | 2.2.2        |
| retriable                                       | 3.1.2        |
| multi_json                                      | 1.15.0       |
| signet                                          | 0.15.0       |
| httpclient                                      | 2.8.3        |
| google-api-client                               | 0.38.0       |
| google-cloud-env                                | 1.5.0        |
| google-cloud-errors                             | 1.1.0        |
| google-cloud-core                               | 1.6.0        |
| google-apis-core                                | 0.3.0        |
| google-apis-storage_v1                          | 0.3.0        |
| digest-crc                                      | 0.6.3        |
| google-cloud-storage                            | 1.31.0       |
| emoji_regex                                     | 3.2.2        |
| uri                                             | 0.10.0       |
| json                                            | 2.5.1        |
| trailblazer-option                              | 0.1.0        |
| declarative                                     | 0.0.20       |
| uber                                            | 0.1.0        |
| representable                                   | 3.1.0        |
| mini_mime                                       | 1.1.0        |
| os                                              | 1.1.1        |
| memoist                                         | 0.16.2       |
| googleauth                                      | 0.16.1       |
| webrick                                         | 1.7.0        |
| rexml                                           | 3.2.5        |
| google-apis-iamcredentials_v1                   | 0.3.0        |
| rake                                            | 13.0.3       |
| aws-eventstream                                 | 1.1.1        |
| aws-sigv4                                       | 1.2.3        |
| aws-partitions                                  | 1.444.0      |
| jmespath                                        | 1.4.0        |
| aws-sdk-core                                    | 3.113.1      |
| aws-sdk-kms                                     | 1.43.0       |
| aws-sdk-s3                                      | 1.93.1       |
| forwardable                                     | 1.3.1        |
| logger                                          | 1.4.2        |
| cgi                                             | 0.1.0        |
| date                                            | 3.0.0        |
| timeout                                         | 0.1.0        |
| stringio                                        | 0.1.0        |
| zlib                                            | 1.1.0        |
| ipaddr                                          | 1.2.2        |
| openssl                                         | 2.1.2        |
| ostruct                                         | 0.2.0        |
| strscan                                         | 1.0.3        |
| delegate                                        | 0.1.0        |
| fileutils                                       | 1.4.1        |
| io-console                                      | 0.5.6        |
| open3                                           | 0.1.0        |
| singleton                                       | 0.1.0        |
| yaml                                            | 0.1.0        |
| psych                                           | 3.1.0        |
| mutex_m                                         | 0.1.0        |
| fastlane-plugin-upgrade_super_old_xcode_project | 0.0.2        |
| fastlane-plugin-huawei_appgallery_connect       | 1.0.15       |
</details>

*generated on:* **2021-04-14**
</details

**Comment by max-ott** [comment]
Did you get any email from Apple that the build was rejected? Is this an update or a brand new app? What is the build / version number?

**Comment by trollderius** [comment]
> Did you get any email from Apple that the build was rejected? Is this an update or a brand new app? What is the build / version number?
That's it. I was using a different email account and I didn't see them.
